runtime/etc/Makefile.launcher

```diff
@@ -62,7 +62,7 @@ else
 # we need to make sure that we are generating position independent code,
 # IE with "-fPIC" on GCC.
 #
-	$(CC) $(GEN_CFLAGS) $(COMP_GEN_CFLAGS) $(SHARED_LIB_CFLAGS) \
+	$(CLIENT_CC) $(GEN_CFLAGS) $(COMP_GEN_CFLAGS) $(SHARED_LIB_CFLAGS) \
```

Comment:
What sets `CLIENT_CC` for non-mli builds?

Reply:
That code path should only run if the build is multilocale, else the existing code for executables is run.

```Makefile

ifneq ($(CHPL_MAKE_IS_MLI),1)
  $(CC) $(GEN_CFLAGS) $(COMP_GEN_CFLAGS) \
    -c -o $(TMPBINNAME)_launcher.o \
    -DLAUNCHER $(CHPL_LN_INCS) -I. $(LAUNCHER_SRC_NAME)
else
#
# If we are building launcher object files for a multi-locale library, then
# we need to make sure that we are generating position independent code,
# IE with "-fPIC" on GCC.
#
  $(CLIENT_CC) $(GEN_CFLAGS) $(COMP_GEN_CFLAGS) $(SHARED_LIB_CFLAGS) \
    -c -o $(TMPBINNAME)_launcher.o \
    -DLAUNCHER $(CHPL_LN_INCS) -I. $(LAUNCHER_SRC_NAME)
endif
```